news/2021-02-04-twc-2021-registrations-open.md

```diff
@@ -0,0 +1,45 @@
+---
+layout: post
+title: "osu!taiko World Cup 2021: Registrations now open!"
+date: 2020-02-04 15:00:00 +0000
+---
+
+Is that time once again, dons and kats being tested into patterns that masters can only hit. The osu!taiko World Cup returns for its eleventh installment! If you think you can join them, the Registration Phase is now open, so sign up today!
+
+![](/wiki/shared/news/banners/TWC_2021.jpg)
+
+For more details, please refer to the **[osu!taiko World Cup 2021 wiki article](/wiki/Tournaments/TWC/2021)**.
+You can discuss this event and follow the most important changes in the **[official discussion thread](https://osu.ppy.sh/community/forums/topics/1241159)**.
+
+## Tournament Schedule
+
+| Event | Timestamp |
+| --: | :-- |
+| Registration Phase | Feb 4th – Feb 18th |
+| Drawings | Feb 27th (14:00 UTC) |
+| Group Stage | Mar 6th & Mar 7th |
+| Round of 16 | Mar 13th & Mar 14th |
+| Quarterfinals | Mar 20th & Mar 21st |
+| Semifinals | Mar 27th & Mar 28th |
+| Finals | Apr 3rd & Apr 4th |
+| Grand Finals | Apr 10th & Apr 11th |
+
+## Prizes
+
+- **1st:** $150 per team member, profile badge, "osu!taiko Champion" user title for one year
+- **2nd:** $80 per team member, profile badge
+- **3rd:** $40 per team member, profile badge
+
+## How do I register?
+
+- **[Click here to sign-up!](https://osu.ppy.sh/community/tournaments/27)**
+- Do not create teams in advance! Teams will be formed by each country's team captain after the Registration Phase has concluded.
```

Comment:
```suggestion
- Do not create teams in advance! Teams will be formed by each country's team captain after the Registration Phase is concluded.
```
or "has been concluded". In any case, passive voice usage seems to be necessary here.